Makefile

```diff
@@ -65,8 +67,10 @@ find_src_files = $(shell find $(dir) -type f -name '*.f90' -o -name '*.c')
 ALL_SRC = $(foreach dir, $(SRC_DIRS), $(find_src_files))
 
 SRC = $(shell $(PY) $(TOOLS)/filter_out.py $(IGNORE) "$(ALL_SRC)")
+SRC += $(CCS_DIR)/src/case_setup/Poisson/poisson_discretisation_kernel.f90
```

Comment:
I'm not sure why this doesn't get added to ALL_SRC - the module it creates isn't used (which would cause cross-compiler problems) so it may not be ignored by makedepf90?